ui/components/app/edit-gas-fee-popover/edit-gas-fee-popover.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import Popover from '../../ui/popover';
+import I18nValue from '../../ui/i18n-value';
+import LoadingHeartBeat from '../../ui/loading-heartbeat';
+
+import EditGasItem from './edit-gas-item';
+
+const EditGasFeePopover = ({ onClose }) => {
+  const t = useI18nContext();
+
+  return (
+    <Popover
+      title={t('editGasFeeModalTitle')}
+      onClose={onClose}
+      className="edit-gas-fee-popover"
+    >
+      <>
+        {process.env.IN_TEST === 'true' ? null : <LoadingHeartBeat />}
+        <div className="edit-gas-fee-popover__wrapper">
```

Comment:
I have seen that you have added the wrapper top border to get the separation line between the title and the rest of the modal, but it seems like the border is cut short a little compared to the design.
Design:
<img width="249" alt="Screen Shot 2021-11-17 at 10 53 44 AM" src="https://user-images.githubusercontent.com/43930900/142234443-93efb0eb-d596-45ba-a96c-1c5038f0b59c.png">
Current work:
<img width="336" alt="Screen Shot 2021-11-17 at 10 54 19 AM" src="https://user-images.githubusercontent.com/43930900/142234548-c5ef9217-56ec-4948-b36c-aca502b9b4f5.png">

Reply:
Yep @NiranjanaBinoy : popup component has some padding in it, so can not avoid that.

Reply:
Can't we override the popover-header styles to get the separation line?

Reply:
I would prefer to be consistent with other popups in the app.